src/zkevm_specs/evm/typing.py

```diff
@@ -204,6 +204,9 @@ def __init__(self, code: Optional[bytearray] = None) -> None:
         self.code = bytearray() if code is None else code
 
     def __getattr__(self, name: str):
+        if name.startswith("bitwise_"):
```

Comment:
notice that on line 212 there is:
```python
opcode = Opcode[name.removesuffix("_").upper()]
```
which allows you to do `Bytecode().not_(a)`

Reply:
good idea. will change it.

Reply:
fixed in 5b9a0d3